addons/mail/static/src/core/common/external_window_service.js

```diff
@@ -0,0 +1,45 @@
+import { registry } from "@web/core/registry";
+import { reactive } from "@odoo/owl";
+
+
+export const externalWindowService = {
+    dependencies: ["ui"],
+    start(env, { ui }) {
+        let externalWindow = null;
+        async function pollClosedWindow() {
+            while (true) {
+                await new Promise((r) => setTimeout(r, 1000));
+                if (externalWindow && externalWindow.closed) {
+                    externalWindow = null;
+                    ui.bus.trigger("resize");
+                    return
+                }
+            }
+        }
+        function getExternalWindow() {
+            if (!externalWindow || externalWindow.closed) {
+                externalWindow = window.open("about:blank", "_blank", "popup=yes");
+                window.addEventListener('beforeunload', () => {
+                    if (externalWindow && !externalWindow.closed) {
+                        externalWindow.close();
+                    }
+                });
+                pollClosedWindow();
+                externalWindow.document.write(window.document.head.outerHTML);
+                externalWindow.document.body = externalWindow.document.createElement('body');
+                ui.bus.trigger("resize");
```

Comment:
> browser.addEventListener("resize", ...);

but triggering a resize event on the ui bus won't trigger a resize event on window... So the event handlers you mention won't be executed... What am I missing?

Reply:
Sorry, I put the wrong listener
https://github.com/odoo/odoo/blob/c4c7f1889d97a45fd7735cc78f448de6aaafd544/addons/web/static/src/views/form/form_controller.js#L159